frontends/p4/simplifyDefUse.cpp

```diff
@@ -177,16 +177,16 @@ class FindUninitialized : public Inspector {
     }
 
     bool preorder(const IR::Function* func) override {
-        LOG3("FU Visiting function " << func->name << "[" << func->id << "]");
+        if (findContext<IR::P4Program>() != nullptr) {
+            LOG3("FU skip function " << func->name << "[" << func->id << "]");
```

Comment:
How can this ever be true?  This pass is only ever called via DoSimplifyDefUse::process, which is only ever called directly on IR::Function, IR::P4Parser, and IR::P4Control nodes and never on full programs...